test/automation/src/sql/notebook.ts

```diff
@@ -60,8 +64,23 @@ export class Notebook {
 
 	// Cell Actions
 
-	async waitForTypeInEditor(text: string) {
-		const editor = '.notebook-cell.active .monaco-editor';
+	async getActiveCell(): Promise<IElement> {
+		const activeCellSelector = '.notebook-cell.active';
+		return this.code.waitForElement(activeCellSelector);
+	}
+
+	async waitForActiveCell(id: string): Promise<void> {
+		const activeCellSelector = '.notebook-cell.active';
+		const activeCell = await this.code.waitForElement(activeCellSelector);
+		if (activeCell.attributes['aria-label'].includes('Code Cell')) {
```

Comment:
Is there a reason you couldn't just wait for element `.notebook-cell.active[id="${id}"]`?